lib/gollum/views/overview.rb

```diff
@@ -4,6 +4,7 @@ module Precious
   module Views
     class Overview < Layout
       attr_reader :results, :ref, :allow_editing, :newable
+      HIDDEN_PATHS = ['.gitkeep']
```

Comment:
This has nothing to do with the issue at hand, but I came across this `'.gitkeep'` string hardcoded in the method below and thought we should perhaps abstract it. Look ok, @bartkamphorst?